test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/RazorGenerateIntegrationTest.cs

```diff
@@ -109,9 +109,6 @@ public class RazorGenerateIntegrationTest : MSBuildIntegrationTestBase
 
             // Act - 2
             // Update the source content and build. We should expect the outputs to be regenerated.
-            // Timestamps on xplat are precise only to a second. Add a delay so we can ensure that MSBuild recognizes the
-            // file change. See https://github.com/dotnet/corefx/issues/26024
-            await Task.Delay(TimeSpan.FromSeconds(1));
```

Comment:
Maybe this is sorted by now

Reply:
Looks like we don't have the fix yet. This test is failing on Travis.

Reply:
Oh well. The issue is marked as fixed, I was kinda hopeful.